src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+    public static final int figure = 5;
+
+    public static Figure getFigure() {
```

Comment:
why methods are static in this class?

Reply:
to access this method in the class

Reply:
you can access not only static methods 
FigureProducer figureProducer = new FigureProducer();
figureProducer.getFigure() 
will work in main mehtod